Documentation/network/bgp-control-plane-troubleshooting.rst

```diff
@@ -0,0 +1,77 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _bgp_control_plane_troubeshooting:
+
+BGP Control Plane Troubleshooting Guide
+=======================================
+
+This document enumerates typical troubles and their solutions when configuring the BGP
+Control Plane.
+
+Even though CiliumBGPPeeringPolicy was applied, the BGP peering is not established
+----------------------------------------------------------------------------------
+
+First, check if the target Node is correctly selected by the
```

Comment:
There's no use of 'second' or 'then', so using 'first' here sticks out. I don't think it's needed:

```suggestion
Check if the target Node is correctly selected by the
```